core/src/main/java/com/yahoo/ycsb/BasicDB.java

```diff
@@ -111,21 +124,23 @@ public Status read(String table, String key, Set<String> fields, HashMap<String,
 
 		if (verbose)
 		{
-			System.out.print("READ "+table+" "+key+" [ ");
+      StringBuilder sb = getStringBuilder();
+			sb.append("READ ").append(table).append(" ").append(key).append(" [ ");
 			if (fields!=null)
 			{
 				for (String f : fields)
 				{
-					System.out.print(f+" ");
+          sb.append(f).append(" ");
```

Comment:
Ah I see. We're rolling all the string ops into SB appends instead of calls to the output buffer.